### DISCUSSION TITLE
Provide prebuilt package?

### DISCUSSION THREAD

**Discussion by xaljer** [jovibor/HexCtrl#61] [Ideas]
That's would be more friendly for new users to try it.

**Comment by jovibor** [comment]
What you mean by package?
Prebuilt `.exe` file?

**Reply by xaljer** [reply]
Yes

**Comment by jovibor** [comment]
You can now download prebuilt package (`.exe`) from the [release page](https://github.com/jovibor/HexCtrl/releases/tag/v3.0.0).
